extensions/interactions/DragAndDropSortInput/directives/DragAndDropSortInput.js

```diff
@@ -46,19 +46,34 @@ oppia.directive('oppiaInteractiveDragAndDropSort', [
             $scope.list.push({title: $scope.choices[i], items: []});
           }
 
+          $scope.treeOptions = {
+            dragMove: function(e) {
+              var elem = document.getElementsByClassName(
```

Comment:
Isn't there way to access this element withour using DOM? Look at the readme of angular-ui.

Reply:
The role of `ui-tree` here is to just detect this event. Else I'm changing the style of the CSS class. I'll take a look.

Reply:
What I meant was that there is placeholder element available in the input event. Check this section (https://github.com/angular-ui-tree/angular-ui-tree/blob/master/README.md#droppedevent)